spec/core_functions/modules/math/abs.hrx

```diff
@@ -0,0 +1,125 @@
+<===>
+================================================================================
+<===> options.yml
+---
+:todo:
+- libsass
```

Comment:
We want to have as much functionality tested on LibSass as possible, so for functions that do have global definitions we should continue to test them globally rather than through `@use`. Similarly, we should keep `spec/core_functions/modules/math.hrx` around until all implementations support `@use` and we can switch the tests to use it by default.

Reply:
How do you feel about letting this new directory thoroughly test the module functions, and keeping `core_functions/math/` around to exercise the global/LibSass functions?

Once everyone supports `@use` we could delete LibSass-specific stuff from `core_functions/math/` and have those specifically for globals.

The con is that we have to maintain both `core_functions/modules/math/` and `core_functions/math/`, but I don't anticipate these functions changing much.

Reply:
(There are several incoming specs for exponential and trig functions that need a home in `modules/math`, and if that directory is inevitable, my personal preference would be to lay the foundation here.)